.github/workflows/run-differential-tests.yaml

```diff
@@ -21,13 +21,23 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Install Dependencies
         run: |
-          curl -o cbmc.deb -L https://github.com/diffblue/cbmc/releases/download/cbmc-5.54.0/ubuntu-20.04-cbmc-5.54.0-Linux.deb
-          sudo apt-get update
-          sudo apt-get install -y --no-install-recommends \
-                                  ./cbmc.deb \
-                                  ninja-build \
-                                  universal-ctags
-          rm -rf cbmc.deb
+          get_latest_release() {
+            curl --silent "https://api.github.com/repos/$1/releases/latest" |
+            jq -r '.tag_name'
+          }
+          cbmc_latest_release=$(get_latest_release diffblue/cbmc)
+          litani_latest_release=$(get_latest_release awslabs/aws-build-accumulator)
+          curl -o cbmc.deb -L \
+            https://github.com/diffblue/cbmc/releases/download/$cbmc_latest_release/ubuntu-20.04-$cbmc_latest_release-Linux.deb
+          curl -o litani.deb -L \
+            https://github.com/awslabs/aws-build-accumulator/releases/download/$litani_latest_release/litani-$litani_latest_release.deb
+          sudo apt-get update \
+            && sudo apt-get install --no-install-recommends --yes \
+                                                      ./cbmc.deb \
+                                                      ./litani.deb \
+                                                      ninja-build \
+                                                      universal-ctags
+          rm -f cbmc.deb litani.deb
           python3 -m pip install jinja2 cbmc-viewer
```

Comment:
It's perhaps too unrelated, but: why is `jinja2` not installed via `apt-get`?

Reply:
No preference. `Jinja2` being a pip package primarily, I installed it with `pip`. Can change if necessary.